src/vms/pvm/etna-builder/builder.ts

```diff
@@ -1327,3 +1321,108 @@ export const newTransferSubnetOwnershipTx: TxBuilderFn<
     addressMaps,
   );
 };
+
+export type NewConvertSubnetTxProps = TxProps<{
+  /**
+   * Specifies which chain the manager is deployed on
+   */
+  chainId: string;
+  /**
+   * Specifies the subnet to be converted
+   */
+  subnetId: string;
+  /**
+   * Specifies the address of the manager
+   */
+  address: Uint8Array;
+  /**
+   * Initial pay-as-you-go validators for the Subnet
+   */
+  validators: readonly Uint8Array[];
+  /**
+   * Indices of existing subnet owners.
+   */
+  subnetAuth: readonly number[];
+}>;
+
+/**
+ * Creates a new unsigned PVM convert subnet transaction
+ * (`NewConvertSubnetTx`) using calculated dynamic fees.
+ *
+ * @param props {NewConvertSubnetTxProps}
+ * @param context {Context}
+ * @returns {UnsignedTx} An UnsignedTx.
+ */
+export const newConvertSubnetTx: TxBuilderFn<NewConvertSubnetTxProps> = (
```

Comment:
Let's add test coverage for this builder.

Reply:
added [here](dab22dd79c8a40f9e85684aa8dae1612c421401f)